### DISCUSSION TITLE
Shared link from iOS to my list shows http 404 in webbrowser on desktop

### DISCUSSION THREAD

**Discussion by mobe2000** [community/community#8134] [Other Feature Feedback, Questions, & Ideas]
👍Thank you for this nice new feature.💚

Obviously I can already use this feature on my iOS App, but can't access the shared link to my list on website in the browser.
https://github.com/stars/mobe2000/lists/homeautomation/ -> leads to http.404
Dont know if I'm already "whitelisted".

**Comment by bailey** [comment]
Hey @mobe2000! Thanks for letting us know. We are still in early days with Lists on the web, so stay tuned as we continue to rollout the feature more broadly. Let us know if you have any feedback on the iOS Lists experience in the meantime ✨
